.circleci/config.yml

```diff
@@ -1,20 +1,54 @@
 version: 2.1
 
+orbs:
+  win: circleci/windows@1.0.0
+
 workflows:
   version: 2
   test:
     jobs:
-      - test-netcore-2-1
+      - test-netcore:
+          name: .NET Core 2.1
+          docker-image: mcr.microsoft.com/dotnet/core/sdk:2.1-focal
+          test-target-framework: netcoreapp2.1
+      - test-netcore:
+          name: .NET Core 3.1
+          docker-image: mcr.microsoft.com/dotnet/core/sdk:3.1-focal
+          test-target-framework: netcoreapp3.1
+      - test-netcore:
+          name: .NET 5.0
```

Comment:
The library itself will be the same .NET Standard 2.0 build in all three of these Linux-based jobs, but since there can be unexpected differences in behavior of the standard runtime library across different platform versions, I've been standardizing the CI jobs for all of our .NET libraries to test them in each of the major Core releases.